src/style.css

```
@@ -0,0 +1,70 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: "Avenir-Book";
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: "Avenir-Heavy";
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+body {
+  margin: 0;
+}
+
+.bar {
+  margin: 20px 8px;
+}
+
+.search {
+  width: 100%;
+  height: 70px;
+  padding: 5px 62px;
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  background-position: 25px 50%;
+  background-size: 20px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  line-height: 22px;
+  font-size: 16px;
+  font-family: "Avenir-Book", sans-serif;
+  font-weight: 300;
+}
+
+.small-class {
+  font-size: 14px;
+  height: 42px;
+  padding: 5px 33px;
+  background-position: 12px 48%;
+  background-size: 11px;
+}
+
+.search:hover {
+  background-color: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.search:focus {
+  background: no-repeat url(images/Search.svg)
+    linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+  font-family: "Avenir-Heavy", sans-serif;
+}
+
+.small-class:hover {
+  background-position: 12px 48%;
+  background-size: 11px;
+}
+
+.small-class:focus {
+  background-position: 12px 48%;
+  background-size: 11px;
+}
```

you're duplicating the code, combine like this:
and no percentages.
```suggestion
.small-class:hover,
.small-class:focus {
  background-position: 12px;
  background-size: 11px;
}
```